src/api/client/resolvers.ts

```diff
@@ -142,27 +132,131 @@ export const queryNodeStitchingResolvers = (
           info
         )
 
-        viewsLookup = createLookup<{ id: string; views: number }>(batchedChannelViews || [])
+        const [batchedChannelFollows, batchedChannelViews] = await Promise.all([
+          batchedChannelFollowsPromise,
+          batchedChannelViewsPromise,
+        ])
+
+        const followsLookup = createLookup<{ id: string; follows: number }>(batchedChannelFollows || [])
+        const viewsLookup = createLookup<{ id: string; views: number }>(batchedChannelViews || [])
+
+        return channels.map((channel: Channel) => ({
+          ...channel,
+          follows: followsLookup[channel.id]?.follows || 0,
+          views: viewsLookup[channel.id]?.views || 0,
+        }))
       } catch (error) {
         SentryLogger.error('Failed to resolve channel views', 'channels resolver', error)
```

Comment:
We should likely update the message here as it will catch both views&follows errors
```suggestion
        SentryLogger.error('Failed to resolve channel views or follows', 'channels resolver', error)
```